man/man5/zfs-module-parameters.5

```diff
@@ -1765,6 +1765,20 @@ table.
 Default value: \fB131,072\fR.
 .RE
 
+.sp
+.ne 2
+.na
+\fBfletcher_4_impl_set\fR (string)
+.ad
+.RS 12n
+Select a fletcher 4 implementation to use.
+Two selectors are supported: \fBfastest\fR and \fBoriginal\fR. If there exists
```

Comment:
if @behlendorf is okay with scalar I can use scalar. Then we have to replace all 'original' to 'scalar' in this patch. We have changed the name many times.

'scalar' and 'avx2' are algorithms but 'fastest' is one of selectors. I'd like to keep it as-is.